### DISCUSSION TITLE
Focus the left sidebar on the current page

### DISCUSSION THREAD

**Discussion by capsia37** [pradyunsg/furo#161] [Ideas] [answered]
Hello! I'd like to suggest a feature for the Furo theme. It would be great if the left sidebar would focus on the currently open page title when we switch page so that we don't have to scroll through the sidebar menu to see where we are and which pages are next.

Thank you!

**Comment by pradyunsg** [comment, marked as answer]
Hi! As a general rule, I've found that users don't like pages that mess with scroll. That said, if someone were to spend some time working on this and coming up with a cross-browser cross-platform way to do this, I'd be happy to accept a PR for this.

**Comment by daltzctr** [comment]
I second this feature suggestion. @pradyunsg I agree with that to an extent. Users don't like pages that mess with *content* scroll. Navigation scroll can and should be fixated on the current content.

**Reply by pradyunsg** [reply]
That's a fair point -- let's move this discussion to https://github.com/pradyunsg/furo/discussions/667.
